packages/electron-scripts/src/commands/build.ts

```diff
@@ -120,6 +125,15 @@ export async function build(options: IBuildCommandOptions): Promise<void> {
 
     const builderConfig = fs.readJsonFileSync(fs.join(basePath, electronBuilderConfigFileName)) as Configuration;
 
+    if (!sign) {
+        // Disable code signing on windows
+        if (builderConfig.win) {
+            Object.assign(builderConfig.win, {certificateSubjectName: null});
```

Comment:
@AlexShemeshWix  Seeing this again, I think we should just allow .js config for builder. Changing process.env and assigning to a readonly field feel too hacky.